scripts/travis_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+RV=1
+
+# Google 2016 Unbreakable
+cd google2016_unbreakable
+python win.py | tee unbreakable.log
+RESULTS=$(grep "CTF{" unbreakable.log)
+if [[ "${RESULTS}" == CTF{0The1Quick2Brown3Fox4Jumped5Over6The7Lazy8Fox9}eeeeeeeeeeeeeeee ]]
```

Comment:
are we sure those e's have to be there at the end and that it's not some artifact from the solver? if we're not sure, maybe just check that the actual key text is there?